FrontEnd/src/main/java/resourceDisplay/LocationObject.java

```diff
@@ -0,0 +1,16 @@
+package resourceDisplay;
+
+/**
+ * This Class is used to get location filter from user
+ */
+public class LocationObject {
```

Comment:
will be making all three into one filterObject in next pr. Not sure if this could be done in thymeleaf because it seems that the set method is used by the template automatically when we use the select.